Sources/AccessibilitySnapshot/SnapshotTesting/SnapshotTesting+Accessibility.swift

```diff
@@ -122,7 +122,18 @@ extension Snapshotting where Value == UIView, Format == UIImage {
             let requiresWindow = (view.window == nil && !(view is UIWindow))
 
             if requiresWindow {
-                let window = UIApplication.shared.keyWindow ?? UIWindow(frame: UIScreen.main.bounds)
+                let keyWindow: UIWindow? = {
+                    if #available(iOS 13.0, *) {
+                        return UIApplication
+                            .shared
+                            .connectedScenes
+                            .flatMap { ($0 as? UIWindowScene)?.windows ?? [] }
+                            .first { $0.isKeyWindow }
+                    } else {
+                        return UIApplication.shared.keyWindow
+                    }
+                }()
+                let window = keyWindow ?? UIWindow(frame: UIScreen.main.bounds)
```

Comment:
Thanks @marcelofabri for the PR 🙌 

it might be worth moving the keyWindow grabbing function somewhere (e.g. an extension of UIApplication, so we only write it once).

Reply:
the main issue is that this extension would need to be declared `public` inside `AccessibilitySnapshot/Core` and I don't think this should be exposed to outside consumers

Reply:
Perhaps a new path [here](https://github.com/cashapp/AccessibilitySnapshot/tree/master/Sources/AccessibilitySnapshot) to include shared files ? Then we could update the podspec/swift package to include that directly as a source to keep it internal.

Reply:
no problem. I will take care of it. Thanks @marcelofabri !